packages/api/src/beacon/routes/debug.ts

```diff
@@ -196,5 +203,15 @@ export function getDefinitions(_config: ChainForkConfig): RouteDefinitions<Endpo
         timeoutMs: 5 * 60 * 1000,
       },
     },
+    getHistoricalSummaries: {
+      url: "/eth/v1/debug/historical_summaries",
```

Comment:
If this endpoint is useful for Portal it makes sense to standardize it on the beacon-api or at least get rough consensus there

if we wanna add a non-standardized api we should rather go with v0 or add to lodestar namespace instead of debug

Reply:
Fair point and this is early days still.  I looked at the Nimbus PR again and it looks like they moved it under a client specific namespace so maybe I should mirror this on the lodestar side for now?

Reply:
Yeah, it would be better to move the api there but I am also fine with v0, although it's not clear if this should be in debug or beacon namespace

Reply:
Understood.  I've switched to v0 for the moment.  I chatted @deme who's working on the Nimbus equivalent and he's hoping to get a PR for the consensus specs to further general conversation there sometime soon.